lms/templates/discussion/index.html

```diff
@@ -22,6 +22,13 @@
 ## Enable fast preview to fix discussion MathJax rendering bug when page first loads.
 <%include file="_js_body_dependencies.html" args="disable_fast_preview=False"/>
 <%static:js group='discussion'/>
+<script language="javascript">
+   $(function () {
+       $("section.discussion").each(function (index, elem) {
+           DiscussionApp.start(elem, true);
+       });
+   });
+ </script>
```

Comment:
What is the purpose of this change? To allow loading this by injecting its HTML into a page after main.js has loaded and the DOM ready event has already fired?